pipelines/analysis_matched.yaml

```diff
@@ -8,6 +8,26 @@ tasks:
       python: |
         from metric_pipeline_tasks import PA1Task
         config.measure.retarget(PA1Task)
+  PA2_design:
+    class: metric_pipeline_tasks.MatchedCatalogAnalysisTask
+    config:
+      connections.package: validate_drp
+      connections.metric: PA2_design
```

Comment:
I need to ask for one more thing.  Can we call these `PA2_design_gri` and `PF1_design_gri` if that's, in fact what they are?  That is now the metrics are named in the metrics package.

Reply:
I'm not sure it makes sense to do it this way, but we should discuss. The way we're measuring them, it is per filter, and the thresholds don't factor into it. So I think that this could be something that is implemented after the fact when comparing to metrics.yaml. 
I'm mostly concerned that adding '_gri' or '_uzy' to the names implies that the calculation includes 3 bands rather than one...